### PR TITLE
refactor:  refactor command policy,payment and crosschain

### PR DESCRIPTION
**Pull request by flywukong**
### Description

1. improve policy cmd API ,  support group policy functions and delete policy function
2. refactor keystore cmd API , improve the usage,  support " gnfd-cmd keystore  generate/inspect"
3. improve config file loading method
4. modify the cmd name to be more clear
5. remove crosschain cmd , change to "object/bucket/group mirror " and "bridge"

### Rationale

make cmd API stable

### Example

NA

### Changes

Notable changes:
* refactor  cmd API
* support new functions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,12 +29,9 @@ The default config file is "config.toml".
 Below is an example of the config file. The rpcAddr and chainId should be consistent with the Greenfield network.
 For Greenfield Testnet, you can refer to [Greenfield Testnet RPC Endpoints](https://greenfield.bnbchain.org/docs/guide/resources.html#rpc-endpoints).
 The rpcAddr indicates the Tendermint RPC address with the port info.
-The configuration for passwordFile is the path to the file containing the password required to generate or parse the keystore.
-Users need to set the password on passwordFile before running commands and the password can be any random string.
 ```
 rpcAddr = "https://gnfd-testnet-fullnode-tendermint-us.bnbchain.org:443"
 chainId = "greenfield_5600-1"
```
Comment (KeefeL) [inline README.md:34]: We should tell user where to save this config file, and in which case this config will be auto generated.

File: README.md [hunk 2]
```diff
@@ -81,13 +77,12 @@ and the passwordFile is used for encrypting/decrypting the private key. The othe
...
+Before generate keystore, you should export your private key from MetaMask and write it into a local file as plaintext.
 
 Assuming that the current private key hex string is written as plaintext in the file key.txt, the following command can be used to generate a keystore file called key.json:
 ```
 // generate keystore key.json
-gnfd-cmd create-keystore --privKeyFile key.txt key.json
+gnfd-cmd create-keystore --privKeyFile key.txt 
 ```
 
 After the keystore file has been generated, you can delete the private key file which contains the plaintext of private key.
```
Comment (KeefeL) [inline README.md:88]: Tell user where the generated keystore file is saved, and also the password file.

File: README.md [hunk 3]
```diff
@@ -230,18 +220,28 @@ gnfd-cmd group head gnfd://groupname
...
+
+#### Quota Operation
+```
+// get quota info
+gnfd-cmd bucket get-quota gnfd://gnfd-bucket
+
+// buy quota
+gnfd-cmd bucket buy-quota --chargedQuota 1000000 gnfd://gnfd-bucket
+```
+
```
Comment (KeefeL) [inline README.md:244]: The following crosschain operation should be modified.

Comment (KeefeL) [reply README.md:244]: We should also add the policy operations

File: README.md (hunk 1 above)
Comment (flywukong) [reply README.md:35]: updated

File: README.md (hunk 2 above)
Comment (flywukong) [reply README.md:96]: updated

**Review by alexgao001 — APPROVED** [review]

**Review by KeefeL — APPROVED** [review]

**Pull request merged**